utils/generate_utils.py

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/python3
+
+# generate_utils.py
+# Date:  24/09/2020
+# Author: Mihai Coșleț
+# Email: coslet.mihai@gmail.com
+
+"""
+Module description
+
+"""
+from pathlib import Path
```

Comment:
is that a top level utils folder? If so, we shall do something about it. Moreover this util looks in fact more as a part of the Report generation functionality. Consider refactoring it please.

Reply:
what do you mean by "top level utils" folder?